builq.go

```diff
@@ -47,16 +51,55 @@ func (b *Builder) Build() (string, []any, error) {
 	return b.query.String(), b.args, b.err
 }
 
+func (b *Builder) writeArgs(s fmt.State, placeholder rune, arg any, isMulti bool) {
+	args := []any{arg}
+	if isMulti {
+		args = b.asSlice(arg)
+	}
+
+	for i, arg := range args {
+		if i > 0 {
+			fmt.Fprint(s, ", ")
+		}
+
+		b.appendArg(arg, placeholder)
+
+		if placeholder == '$' { // PostgreSQL
+			fmt.Fprintf(s, "$%d", b.counter)
+		} else { // MySQL/SQLite
```

Comment:
Should probably check for `?` explicitly to be consistent with the `switch` above

Reply:
done